postgres/assets/dashboards/postgresql_screenboard_dashboard.json

```
@@ -3,895 +3,1381 @@
-        "background_color": "gray",
-        "font_size": "14",
-        "text_align": "left",
-        "show_tick": true,
-        "tick_pos": "50%",
-        "tick_edge": "top"
       }
     },
     {
-      "id": 8293335103140496,
-      "layout": {
-        "x": 33,
-        "y": 84,
-        "width": 70,
-        "height": 5
-      },
+      "id": 1127561260333130,
       "definition": {
-        "type": "note",
-        "content": "Locks",
+        "title": "Logs",
+        "title_align": "center",
+        "type": "group",
         "background_color": "blue",
-        "font_size": "18",
-        "text_align": "center",
-        "show_tick": false,
-        "tick_pos": "50%",
-        "tick_edge": "bottom"
-      }
-    },
-    {
-      "id": 8581080208165878,
-      "layout": {
-        "x": 177,
-        "y": 7,
-        "width": 47,
-        "height": 55
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 8685226852632656,
+            "definition": {
+              "title": "",
+              "title_size": "16",
+              "title_align": "left",
+              "type": "log_stream",
+              "indexes": [],
+              "query": "source:postgresql",
+              "sort": {
+                "column": "time",
+                "order": "desc"
+              },
+              "columns": [
+                "core_host",
+                "core_service"
+              ],
+              "show_date_column": true,
+              "show_message_column": true,
+              "message_display": "expanded-md"
+            },
+            "layout": {
+              "x": 0,
+              "y": 0,
+              "width": 6,
+              "height": 4
+            }
+          }
+        ]
       },
-      "definition": {
-        "title": "",
-        "title_size": "16",
-        "title_align": "left",
-        "type": "log_stream",
-        "indexes": [],
-        "query": "source:postgresql",
-        "sort": {
-          "column": "time",
-          "order": "desc"
-        },
-        "columns": [
-          "core_host",
-          "core_service"
-        ],
-        "show_date_column": true,
-        "show_message_column": true,
-        "message_display": "expanded-md"
-      }
-    },
-    {
-      "id": 7175452850503336,
       "layout": {
-        "x": 177,
-        "y": 1,
-        "width": 47,
+        "x": 6,
+        "y": 0,
+        "width": 6,
         "height": 5
-      },
-      "definition": {
-        "type": "note",
-        "content": "Logs",
-        "background_color": "blue",
-        "font_size": "18",
-        "text_align": "center",
-        "show_tick": false,
-        "tick_pos": "50%",
-        "tick_edge": "bottom"
       }
     }
   ],
   "template_variables": [
     {
       "name": "scope",
-      "default": "*",
-      "prefix": null
+      "default": "*"
     }
   ],
-  "layout_type": "free",
-  "is_read_only": true,
+  "layout_type": "ordered",
+  "is_read_only": false,
```

I think this should be set to `true` as integration dashboards are read only. :) 

```suggestion
  "is_read_only": true,
```